ExpenseTracker.Core/Models/Transactions/Exceptions/NullTransactionException.cs

```diff
@@ -11,5 +11,8 @@ public class NullTransactionException : Xeption
     {
         public NullTransactionException() : base(message: "Transaction is null.")
         { }
+
+        public NullTransactionException(string message) : base(message: message)
```

Comment:
```suggestion
        public NullTransactionException(string message)
            : base(message: message)
```

Reply:
General rule of good coding is to be consistent in your classes, if you break them in some do it for all.

Reply:
Fixed.